src/Read.jl

```diff
@@ -0,0 +1,167 @@
+const ZIP_FILE_HEADER = UInt8[ 0x50, 0x4b, 0x03, 0x04 ] # valid Zip file header
+const ns = "http://schemas.openxmlformats.org/spreadsheetml/2006/main"
+
+function _check_file_format(byte_array::Vector{UInt8})
+    header = @view(byte_array[1:4])
+    header == ZIP_FILE_HEADER || error("WrongExtension: supported only .xlsx format")
+    return nothing
+end
+
+function _read_xml_by_name(zip::ZipFile.Reader, name::String)
+    xml_doc = nothing
+    for f in zip.files
+        f.name == name || continue
+        xml_doc = EzXML.parsexml(read(f))
+        break
+    end
+    return xml_doc
+end
+
+function _get_shared_strings(ss_doc::Union{Nothing, EzXML.Document})
+    shared_strings = String[]
+    isnothing(ss_doc) && return shared_strings
+
+    for si in EzXML.findall("//x:si", ss_doc.root, ["x"=>ns])
+        str_content = ""
+
+        for node in EzXML.eachelement(si)
+
+            if EzXML.nodename(node) == "t"
+                str_content *= EzXML.nodecontent(node)
+            elseif EzXML.nodename(node) == "r"
+                for r_node in EzXML.eachelement(node)
+
+                    if EzXML.nodename(r_node) == "t"
+                        str_content *= EzXML.nodecontent(r_node)
+                    end
+                end
+            end
+        end
+        push!(shared_strings, str_content)
+    end
+
+    return shared_strings
+end
+
+function _find_t_node_recursively(node::EzXML.Node)
+    if EzXML.nodename(node) == "t"
+        return node
+    else
+        for child in EzXML.eachelement(node)
+            result = _find_t_node_recursively(child)
+            if result != nothing
+                return result
+            end
+        end
+    end
+
+    return nothing
+end
+
+function _parse_cell(cell::EzXML.Node, shared_strings::Vector{String})
+    t = haskey(cell, "t") ? cell["t"] : ""
+
+    for child_elem in EzXML.eachelement(cell)
+        child_name = EzXML.nodename(child_elem)
+
+        if t == "inlineStr"
+
+            if child_name == "is"
+                t_node = _find_t_node_recursively(child_elem)
+                if t_node != nothing
```

Comment:
У меня линтер на такое ругается 
```suggestion
                if !isnothing(t_node)
```

Reply:
Ой какой ужас! Конечно только `isnothing`! Не знаю что на меня нашло)